superset/cccs/pythonpath/jinja_context_addons.py

```diff
@@ -118,17 +153,33 @@ def render_ipv4_either_number_columns(src_col, dst_col, val):
     # ( SRC_IP = 22345 OR (SRC_IP >= 10000 AND SRC_IP <= 20000) )
     # OR
     # ( DST_IP = 22345 OR (DST_IP >= 10000 AND DST_IP <= 20000) )
-    return ' OR '.join(render_in_conditions(val, src_col)
-                     + render_in_conditions(val, dst_col))
-    
+    return " OR ".join(
+        render_in_conditions(val, src_col) + render_in_conditions(val, dst_col)
+    )
 
 
+def dashobard_link(link_label, dashboard_id, src_column, target_column):
```

Comment:
I'd suggest putting a return type here, but I'm not even sure what the return type is. Str?